docs/structure-samples/python/python.md

```diff
@@ -0,0 +1,50 @@
+## Python project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a Python standard application.
+
+```bash
+
+~root
+│
+├── .devcontainer.json              
+├── azure.yaml                      
+│
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml           
+│       ├── workflow2.yml           
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep                  
+│   ├── main.parameters.bicep       
+│   ├── abbreviations.json          
+│   └── ...                         
+│
+├── src/                            
+│   ├── projectname/                
+│   │   ├── __init__.py             
+│   │   ├── core/                   
+│   │   ├── models/                 
+│   │   ├── services/               
+│   │   └── ...
+│   └── scripts/                    
+│
+├── tests/                          
+│   └── ...                         
+│
+├── *docs/                           
+│   └── ...                         
+│
+├── requirements.txt                
```

Comment:
```suggestion
├── requirements-dev.txt                
```

If there are requirements at this level, they'd probably be developer specific requirements. Typically the project requirements are inside src or projectname.